voice-agent-openid-provider/config/registry/registry-incl.xml

```diff
@@ -13,7 +13,7 @@
 <server>
    <basicRegistry id="basic" realm="OpBasicRealm">
       <!-- replace / add user -->
-      <user name="bpulito" password="{hash}ATAAAAAI7tXy93T2DDdAAAAAIEny+m0RVHlDl2o+hkRcpihcpogPTuI6QznzUMHzYmKd" />
+      <user name="username" password="{password hash" />
```

Comment:
Weird hash replacement happened in this file, shouldn't this be 

```
password="{hash} hashed_password" 
```

Reply:
Not sure what happened with that edit. Thanks.